.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
+---
+
+_When issuing a bug report please include as many of the below details as possible. If you find a Closed issue that seems like it is the same thing that you're experiencing, open a new issue and include a link to the original issue in the body of your new one._
+
+**Describe the bug**
```

Comment:
Do we want:

# Describe the bug

Using header tags?

https://github.com/stevemao/github-issue-templates/blob/master/bugs-only/ISSUE_TEMPLATE.md

Reply:
Maybe not H1 though 😸

Reply:
done